exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/join/JoinUtils.java

```diff
@@ -65,6 +68,11 @@
   }
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(JoinUtils.class);
 
+  public static final String FAILED_TO_PLAN_CARTESIAN_JOIN = String.format(
+      "This query cannot be planned possibly due to either a cartesian join or an inequality join. " +
+          "If cartesian or inequality join is used intentionally, set option '%s' to false and try again.",
```

Comment:
"If a cartesian" instead of "If cartesian" and "set the option" instead of "set option"??